examples/illustrations/existence_intervals/README.md

```diff
@@ -115,3 +115,101 @@ Some of the examples in the OWL-TIME specification include demonstrations of `ti
 * [5.4, "iCalendar"](https://www.w3.org/TR/owl-time/#iCal) instantiates the endurant "Abraham Lincoln" (node `_:TE-2`).
 * [5.6, "A Use Case for Scheduling"](https://www.w3.org/TR/owl-time/#scheduling) instantiates perdurants some teleconference and some meeting (nodes `ex:telecon` and `ex:meeting`, respectively).
 * [5.7, "Alignment of PROV-O with OWL-Time"](https://www.w3.org/TR/owl-time/#time-prov) states that `prov:Activity` and `prov:InstantaneousEvent` can be subclasses of `time:TemporalEntity` and `time:Instant`, respectively.
+
+
+## gUFO
+
+(gUFO version: [1.0.0](https://github.com/nemo-ufes/gufo/releases/tag/v1.0.0).)
+
+gUFO's topmost class that describes an object that exists in a period of time is "Concrete Individual" (`gufo:ConcreteIndividual`), defined as the disjoint union of "Endurants," "Events," and "Situations."
+
+* `gufo:Endurant` corresponds with UCO Issue 535's "Endurant".
+* `gufo:Event` corresponds with UCO Issue 535's "Perdurant".
+* `gufo:Situation` is left out of scope of this discussion.
+
+Two properties, "has begin point" and "has end point," assign the first and last instants that any concrete individual exists.  The properties have range `time:Instant`, inheriting the vocabulary and entailing the structures of OWL-Time as described above.
+
+None of the three subclasses of `gufo:ConcreteIndividual` specialize restrictions on those existence-bounding properties.  Hence, without loss of generality, we can review a `gufo:Endurant` and see the same time-related applicability for `gufo:Event`.
+
+This is an endurant (more specifically, a `gufo:Object`) with literal-valued instants describing the creation and termination timestamps TS0 and TS1:
+
+```json
+[
+    {
+        "@id": "kb:Object-bee97e78-8e5e-4ffe-9c2b-28e960f84bbc",
+        "@type": "gufo:Object",
+        "rdfs:label": "T",
+        "rdfs:comment": "A thing with a time-bounded existence.",
+        "gufo:hasBeginPointInXSDDateTimeStamp": {
+            "@type": "xsd:dateTimeStamp",
+            "@value": "2020-01-02T03:04:05.6789Z"
+        },
+        "gufo:hasEndPointInXSDDateTimeStamp": {
+            "@type": "xsd:dateTimeStamp",
+            "@value": "2021-02-03T04:05:06.7890Z"
+        }
+    }
+]
+```
+
+This is the same endurant with reified instants housing the beginning and end:
+
+```json
+[
+    {
+        "@id": "kb:Object-bee97e78-8e5e-4ffe-9c2b-28e960f84bbc",
+        "rdfs:label": "T",
```

Comment:
I'd suggest to include here:
`"@type": "gufo:Object"`

Reply:
Done in `cc10413`.